pkgs/development/python-modules/confusable-homoglyphs/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  unittestCheckHook,
+  wheel,
+  click,
+}:
+
+let
+  kebabToSnake = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+in
+buildPythonPackage rec {
+  pname = "confusable-homoglyphs";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = kebabToSnake pname;
+    hash = "sha256-uZUAHJsuG0zqDPXzhAp8eRiKjLutBT1pNXK9jBwexGA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  disabled = pythonOlder "3.7";
+
+  optional-dependencies = {
+    cli = [ click ];
+  };
+
+  checkInputs = [ unittestCheckHook ] ++ optional-dependencies.cli;
+
+  pythonImportsCheck = [ (kebabToSnake pname) ];
+
+  unittestFlagsArray = [
+    "-s"
+    "tests"
+    "-v"
+  ];
+
+  meta =
+    let
+      inherit (lib) licenses maintainers;
+    in
+    {
+      description = "Detect confusable usage of unicode homoglyphs, prevent homograph attacks.";
```

```suggestion
      description = "Detect confusable usage of unicode homoglyphs, prevent homograph attacks";
```
> Not end with a period (or any punctuation for that matter).

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

---

thanks!